article/Future-work/future-work.tex

```diff
@@ -1 +1,14 @@
-\section{Future work}
\ No newline at end of file
+\section{Future work}
+We have several ideas for future work.
+There are obviously other methods to calculate the weighted summation layer combinations that could be investigated.
+Additionally these layer combinations could be based on information about the dataset, so that it is not necessary to run each layer individually.
+These parameters in layer combination could also be parameters that were to be learned.
+We tried adding one weight to LightGCN and used different layer combinations in Appendix \ref{app:lightgcn-1-weight} to see if it with this way could learn the layer combination, but this method did not show improvements.
```

Comment:
vi tilføjede vel ikke kune en weight ? hvad det ikke en matrice af weights ?
Vi bliver ved med at snakke om "the layer combination" igennem hele paperet, men det er sgu lidt en mærkelig formulering. Hvad er "The layer combination" ? er det hvilken aggreagtion function man bruger ? tror vi lige skal snakke om det i morgen of finde en lidt mere formel term til det.